source/acquaman/BioXAS/BioXASXASScanConfiguration.h

```diff
@@ -26,7 +26,7 @@ class BioXASXASScanConfiguration : public AMGenericStepScanConfiguration, public
 	virtual ~BioXASXASScanConfiguration();
 
 	/// Returns a description of the scan technique.
-	virtual QString technique() const { return "XAS"; }
+	virtual QString technique() const { return "XAS Scan"; }
```

Comment:
There is no requirement to have "Scan" in them. I saw the majority have "Scan" in the string. Therefore, I just make sure everyone has it.